overthrower/overthrower.cpp

```diff
@@ -166,15 +166,15 @@ static unsigned int readValFromEnvVar(const char* env_var_name, const unsigned i
 
 extern "C" void activateOverthrower()
 {
-#if defined(__APPLE__)
+//#if defined(__APPLE__)
```

Comment:
Please restore `#if`, if it is not necessary on Linux.
Or remove `#if` if it helps on Linux.